doc/admin/admin_commands/kdb5_ldap_util.rst

```diff
@@ -252,7 +250,9 @@ list
 
     **list**
 
-Lists the name of realms.
+Lists the name of realms under the container.  If not overridden by
+command-line options, the LDAP server information for the default
+realm is used to search the container.
 
```

Comment:
The synopsis of kdb5_ldap_util was updated, so that -r REALM can be passed to all commands and the lack of it implies the default realm.  This means, that “kdb5_ldap_util list -r X.NET” will use the LDAP settings for the X.NET realm and “kdb5_ldap_util list” will use the LDAP settings of the default realm.  Before definigng a global `-r` this was not so obvious, and ignoring the ldap_servers in [dbdefaults] made things even more confusing.  In my experiment I had ldap_servers in [dbdefault] and the default realm has not used LDAP backend.

I did ask to document for `list` where it obtains its settings for the LDAP connection from.  Provided that a missing `-r` implies settings for the default realm, I think now that `If not overridden by command-line options, the LDAP server information for the default realm is used to search the container.` is redundant.

Reply:
Okay, I'll take it out.